widgetssdk/src/main/java/com/glia/widgets/view/unifiedui/Merge.kt

```diff
@@ -1,10 +1,14 @@
 package com.glia.widgets.view.unifiedui
 
-import kotlin.reflect.KClass
-import kotlin.reflect.KParameter
-import kotlin.reflect.KProperty1
-import kotlin.reflect.full.declaredMemberProperties
-import kotlin.reflect.full.primaryConstructor
+/**
+ * Interface for classes that can be merged with another instance of the same class
```

Comment:
Lets maybe add that it is for data classes?
with another instance of the same class => with another instance of the same **data** class

Reply:
There is no restriction on the usage of any other classes, which is why I preferred to not mention the `data` class here.